### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by fsimonis**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+about: Report a bug you noticed.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe your setup**
+
+Distribution: _Ubuntu 18.04_
```
Comment (MakisH) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Operating system (e.g. Linux distribution and version): _Ubuntu 18.04_
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+preCICE Version: x.y.z
+
+Add the versions of further adapter and/or bindings.
```
Comment (MakisH) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Add the versions/commit of further adapter and/or bindings.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+**Describe the problem**
+Describe the problem in detail including error output, logs, screenshots etc.
+Please attach relevant files to this issue.
+
+**Step To Reproduce**
+1. Run this command
+2. than this command
```
Comment (MakisH) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
2. then this command
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+**Expected behaviour**
+A clear and concise description of what you expected to happen.
+
+**Additional context**
+Add any other context about the problem here.
+Attach relevant logs and files to this issue.
```
Comment (MakisH) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Attach relevant logs and files to this issue.
Format your text (and especially any output/error messages) (see the [GitHub Markdown guide](https://guides.github.com/features/mastering-markdown/)
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Report a bug you noticed.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
```
Comment (MakisH) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion

If you found a clear bug in the preCICE library itself, please use this issue. You can also open issues in [preCICE adapters and tutorials](https://github.com/precice).
For general user support, please use the [Discourse forum](https://precice.discourse.group/) or the [Gitter chatroom](https://gitter.im/precice/Lobby).
```

**Comment by fsimonis** [comment]
I just opened an issue following this template. It takes longer, but it forces you to think a bit more about the setup. 
https://github.com/precice/precice/issues/684

**Pull request merged**